Automattic/wp-calypso
# Update tour events to include `editor_type`

## Proposed Changes

Update tour events in the editor to include `editor_type` (in this case, site | post | page).

Because there are future analyses we can do where we know a user interacts more with Tour in certain cases, in the case of write flow or using the Assembler.

Part of https://github.com/Automattic/wp-calypso/issues/73142
Follow up for https://github.com/Automattic/wp-calypso/pull/73565

## Testing Instructions



- Open `/site-editor/<your-site>`
- Open Welcome Guide
- Verify events with [Tracks Vigilante Chrome Extension](https://github.com/Automattic/tracks-chrome-extension)

<img width="756" alt="Screen Shot 2023-02-27 at 17 15 43" src="https://user-images.githubusercontent.com/5287479/221509802-67b89201-e662-46de-b254-cd08eaca7cc4.png">

- Open `/post/<your-site>/<site-id>`
- Open Welcome Guide
- Verify events with [Tracks Vigilante Chrome Extension](https://github.com/Automattic/tracks-chrome-extension)
<img width="763" alt="Screen Shot 2023-02-27 at 17 18 11" src="https://user-images.githubusercontent.com/5287479/221510230-4aa3cca2-910e-42d8-8cf4-ca722bf04f5a.png">

- Open `/page/<your-site>/<site-id>`
- Open Welcome Guide
- Verify events with [Tracks Vigilante Chrome Extension](https://github.com/Automattic/tracks-chrome-extension)

<img width="757" alt="Screen Shot 2023-02-27 at 17 17 29" src="https://user-images.githubusercontent.com/5287479/221510095-20396455-f391-4110-8643-ec67ba3c45a1.png">

## Pre-merge Checklist
